bitnami/metallb/templates/speaker/daemonset.yaml

```
@@ -35,6 +35,10 @@ spec:
       serviceAccountName: {{ include "metallb.speakerServiceAccountName" . }}
       terminationGracePeriodSeconds: {{ .Values.speaker.daemonset.terminationGracePeriodSeconds }}
       hostNetwork: true
+      {{- if gt (len .Values.speaker.initContainers) 0 }}
```

I think it should be enought with the following:
```suggestion
      {{- if .Values.speaker.initContainers }}
```